### DISCUSSION TITLE
How i can send sticker?

### DISCUSSION THREAD

**Discussion by ivanwebdevs** [WhiskeySockets/Baileys#21] [General]
its possible to send sticker with png or jpg file from url use baileys?
if it is possible please make an example for me 😲

**Comment by FaizBastomi** [comment]
Sending stickers in png or jpg format is not possible, due to the limitations of the WhatsApp application which only supports the webp format

**Reply by ivanwebdevs** [reply]
ohh thanks bro can you give me example request to send stickers?

**Reply by andresayac** [reply]
I use this https://www.npmjs.com/package/wa-sticker-formatter
an example i found [Example](https://github.com/codigoencasa/bot-whatsapp/blob/c9cdd393a0533282438ece011e2e0195717b56d0/packages/provider/src/baileys/index.js#L396)

or send image webp

```
const id = 'abcd@s.whatsapp.net' // the WhatsApp ID 
const msgPoll = {
    sticker: {
        url: "https://dayax.net/Media/sticker_mv.webp"
    }
};
sock.sendMessage(id, msgPoll)
```

**Reply by ivanwebdevs** [reply]
> I use this https://www.npmjs.com/package/wa-sticker-formatter an example i found [Example](https://github.com/codigoencasa/bot-whatsapp/blob/c9cdd393a0533282438ece011e2e0195717b56d0/packages/provider/src/baileys/index.js#L396)
> 
> or send image webp
> 
> ```
> const id = 'abcd@s.whatsapp.net' // the WhatsApp ID 
> const msgPoll = {
>     sticker: {
>         url: "https://dayax.net/Media/sticker_mv.webp"
>     }
> };
> sock.sendMessage(id, msgPoll)
> ```

ohh thankss
